### DISCUSSION TITLE
Best practices for TV show dialogues

### DISCUSSION THREAD

**Discussion by YashvardhanRanawat7** [MaartenGr/BERTopic#2159] [General]
Hey all, 

I am trying to create a topic model of an episode from my favorite TV show, So far I have used the best practices mentioned in the documentation and also using Llama 2 to fine tune topics 

Is there anything else I can do to improve results?

**Comment by MaartenGr** [comment]
Thank you for reaching out. It's difficult to say what you can improve if I do not know the results nor the code itself. Having said that, what are you looking to improve? If you want to improve the LLM output, then you could use LLama 3.1 instead for example.
